nodedb.go

```diff
@@ -497,7 +503,7 @@ func (ndb *nodeDB) deleteVersion(version int64, cache *rootkeyCache) error {
 
 	// check if the version is referred by the next version
 	nextRootKey, err := cache.getRootKey(ndb, version+1)
-	if err != nil {
+	if err != nil && err != ErrVersionDoesNotExist {
```

Comment:
Could `nextRootKey` be `nil` above if `ErrVersionDoesNotExist`?

Reply:
Yeah it can if both the current version and the next version are missing

Reply:
we don't have a check for it being nil right?